### DISCUSSION TITLE
A reason of being

### DISCUSSION THREAD

**Discussion by eltikia** [eltikia/Perl-1.0#50] [Q&A] [answered]
Is there any reason of being for this project? Can you tell us what is it all about?

**Comment by Stagyrite** [comment, marked as answer]
I'd call developing the Perl 1 compiler my historical approach to the Perl programming language. I want to learn Perl and Raku (Perl 6), and therefore I have to choose a way. These are the personal reasons for the project. Its reason of being is that compilers should be fresh (e.g., themselves compiling). It's also about something one can call poetry, and I mean poetry when it provides a quick result. It would certainly take longer to refresh some other projects (e.g. [Linux kernel ver. 0.01](https://github.com/zavg/linux-0.01)). In Raku, one can get a result that takes fewer lines of code than in Java, as one can see in [a separate discussion](https://github.com/eltikia/Perl-1.0/discussions/45).
